src/core_modules/capture-core/components/Pages/Search/SearchForm/SearchForm.container.js

```diff
@@ -84,5 +87,8 @@ const mapDispatchToProps = (dispatch: ReduxDispatch): DispatchersFromRedux => ({
 });
 
 
-export const SearchForm =
-  connect<Props, OwnProps, _, _, _, _>(mapStateToProps, mapDispatchToProps)(SearchFormComponent);
+export const SearchForm: ComponentType<OwnProps> =
+  compose(
+      connect<Props, OwnProps & CssClasses, _, _, _, _>(mapStateToProps, mapDispatchToProps),
+      withStyles(getStyles),
```

Comment:
Happy to discuss this if you find it strange. To me it looks fine but I can see why one might not like it

Reply:
Hehe, you're right, I don't like it. Some of the reasons:

1. The component file should export when thing only, the component.
2. I feel adding styles to the component shouldn't be a concern for the container file, at least not when getStyles is defined in the component file in the first place. It feels right to keep this internal to the component file.
3. We're changing the code to accommodate for flow.

Reply:
I see your points. Fair 👍